packages/desktop-client/src/components/reports/CustomTick.tsx

```diff
@@ -0,0 +1,9 @@
+const CustomTick = (value: string, test: boolean) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏this isn't a component. It's a function for retrieving the tick values. As such: it should follow `camelCase` and the filename should also be updated from `tsx` to `ts`.

```suggestion
const getCustomTick = (value: string, test: boolean) => {
```